.bazelrc

```diff
@@ -1,3 +1,3 @@
 # We can't create a bzl_library for rules-swift because of its visibility,
 # so circumvent by not using the sandbox
-build --strategy=Stardoc=standalone
+common --strategy=Stardoc=standalone
```

Comment:
I see the standalone value listed in the bazel manual for the strategy option. But why is Stardoc also mentioned here?